### PR TITLE
Yoroi short term improvement pt.1 

### PR DESCRIPTION
**Pull request by neuodev**
# Changes 
1. [Change Summary Page Layout](https://emurgo.atlassian.net/jira/software/c/projects/YOEXT/boards/9?modal=detail&selectedIssue=YOEXT-154)
2. [Add skeleton loader to the dashboard page](https://emurgo.atlassian.net/jira/software/c/projects/YOEXT/boards/9?modal=detail&selectedIssue=YOEXT-153)
3. [Add the Support button ](https://emurgo.atlassian.net/jira/software/c/projects/YOEXT/boards/9?modal=detail&selectedIssue=YOEXT-155)
4. [Add FAQ external button to the left side navigation](https://emurgo.atlassian.net/jira/software/c/projects/YOEXT/boards/9?modal=detail&selectedIssue=YOEXT-156)

# Screenshots 
Screen Shot 2022-05-24 at 1 34 11 PM

Screen Shot 2022-05-24 at 1 34 35 PM

### CODE REVIEW

**Review by Nebyt — APPROVED** [review]
:superhero:

File: packages/yoroi-extension/app/components/widgets/Support.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+      const script = document.createElement('script')
+      script.src = src
+      script.id = id
+      document.body?.appendChild(script)
+  }
+
+  componentDidMount() {
+      this.loadScript('https://static.zdassets.com/ekr/snippet.js?key=68b95d72-6354-4343-8a64-427979a6f5d6', 'ze-snippet');
+
+      const interval = setInterval(()=>{
```
Comment (yushih) [inline packages/yoroi-extension/app/components/widgets/Support.js:21]: Why not use setTimeout?

File: packages/yoroi-extension/app/styles/context/layout.js [hunk 2]
```diff
@@ -2,6 +2,7 @@
 import React from 'react';
 import type { Node } from 'react';
 import { THEMES } from '../utils';
+// import { ReactComponent as SupportIcon } from '../../assets/images/support.svg'
```
Comment (yushih) [inline packages/yoroi-extension/app/styles/context/layout.js]: Please remove this change

File: packages/yoroi-extension/app/components/widgets/Support.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+type Props = {||}
+
+export default class Support extends Component <Props> {
+
+  loadScript(src: string, id: string): void {
+      const script = document.createElement('script')
+      script.src = src
+      script.id = id
+      document.body?.appendChild(script)
+  }
```
Comment (yushih) [inline packages/yoroi-extension/app/components/widgets/Support.js:16]: No. We can't do this. We are a wallet!

File: packages/yoroi-extension/app/components/topbar/NavWalletDetails.js [hunk 4]
```diff
@@ -36,6 +38,7 @@ type Props = {|
   +showDetails?: boolean,
   +getTokenInfo: $ReadOnly<Inexact<TokenLookupKey>> => $ReadOnly<TokenRow>,
   +defaultToken: $ReadOnly<TokenRow>,
+  +purpose?: 'allWallets' | 'topBar',
```
Comment (yushih) [inline packages/yoroi-extension/app/components/topbar/NavWalletDetails.js]: Please make this non-optional and do not relying on default value.

**Review by yushih — CHANGES_REQUESTED** [review]
Please change Support.js so that no script is loaded from outside source.  When the support button is pressed, pop up an iframe with the contents from zendesk.

File: packages/yoroi-extension/app/components/widgets/Support.js (hunk 1 above)
Comment (yushih) [reply packages/yoroi-extension/app/components/widgets/Support.js:21]: Never mind. We shouldn't do this anyway.

Comment (neuodev) [reply packages/yoroi-extension/app/components/widgets/Support.js:21]: Normally when the Zendesk script got loaded the **popup** opens automatically. to disable this behavior I need to check and hide it.

**Comment by neuodev** [comment]
@yushih  I fixed all of the required changes so far except for loading the Zendesk JS script.

**Review by yushih — APPROVED** [review]

**Pull request merged**